src-django/api/models.py

```diff
@@ -68,3 +69,30 @@ def save(self, **kwargs):
     class Meta:
         app_label = 'api'
         ordering = ['page', 'display_index']
+
+class Concept(models.Model):
+    TYPES = (
+        ('string', 'string'),
```

Comment:
because it was in Element ¯\_(ツ)_/¯

On Wednesday, 20 January 2016, Stephen Li <notifications@github.com> wrote:

> In src-django/api/models.py
> <https://github.com/SanaMobile/sana.protocol_builder/pull/297#discussion_r50324729>
> :
>
> > @@ -68,3 +69,30 @@ def save(self, **kwargs):
> >      class Meta:
> >          app_label = 'api'
> >          ordering = ['page', 'display_index']
> > +
> > +class Concept(models.Model):
> > +    TYPES = (
> > +        ('string', 'string'),
>
> why is this a pair?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SanaMobile/sana.protocol_builder/pull/297/files#r50324729>
> .
>